django_slack/templates/django_slack/exception

```diff
@@ -0,0 +1,5 @@
+{% extends django_slack %}
```

Comment:
is this right? Look at the other templates, they are just ``{% block blah %}{% endblock %}``

(This is really another way of saying, please comment this exception!)